safetrace-sdk/Services/Networking/Networking.swift

```diff
@@ -171,6 +175,7 @@ class Network: NetworkProtocol {
         struct HealthCheckPayload: Encodable {
             let bluetooth_enabled: Bool
             let notifications_enabled: Bool
+            let location_enabled: Bool
```

Comment:
Adding a new `location_enabled` value to health check for location permissions, FYI @enyuhsu